docs/dogfood.md

```diff
@@ -9,6 +9,8 @@ as gittuf itself may have breaking changes in the coming months. After gittuf
 reaches v1, we expect to reset the policy and start over with a formal root
 signing. We envision dogfooding to happen in several phases.
 
+**NOTE: when using gittuf with any server, to push and pull gittuf specific refs (e.g RSL, Policy, and Attestations), `git push remote-name refs/gittuf/*` and `git fetch remote-name refs/gittuf/*:refs/gittuf/*` must be used instead of gittuf commands for the v.0.4.0 relese**
```

Comment:
This sort of comes out of nowhere right now. I'd make this a whole section below titled "Verifying gittuf using gittuf", which would include instructions for using either the gittuf CLI to clone the repo and verify, or the git CLI.

Reply:
Also, please wrap at 80 cols!

Reply:
Wrapped, and I have created a new section